### PR TITLE
Migrate pipeline

### PR DESCRIPTION
**Pull request by dchiu911**
- Renamed histotypes to updated labels: HGSC -> HGSOC, LGSC -> LGSOC, MUC -> MUOC. This was done in all the input data (training set, confirmation set, validation set). It avoids having to relabel the previous outputs that used the old labels in multiple places of the analysis
- Reran whole pipeline. The new "general" queue on the GSC server was changed to be the default partition as of March 12, 2026. It uses a newer OS called Rocky 9 as opposed to CentOS 7. The old "upgrade" queue will be deprecated. In anticipation of reproducibility issues, I migrated the whole pipeline to use the general queue, and also updated the R version used from 4.4.2 to 4.5.2.
- All bioconductor packages and some CRAN packages needed to be updated as a result from the R version update. Most of these changes did not have an impact on the results, except for xgboost. I strongly believe the most recent version of xgboost is better than the previous one with regards to the model specifications.
- The rule for choosing the optimal number of genes during gene optimization is now simpler. It used to be the fewest number of genes needed to achieve a mean F1-score > 0.7. We can define a simpler rule now, given how the distribution of F1-scores is not just monotonically increasing by number of genes. The rule is: the number of genes at which the mean F1-score is the highest. 
- SMOTE-SVM has replaced Up-XGB among the top 5 workflows. The final workflow is now also Hybrid-XGB instead of SMOTE-SVM. Manuscript results, tables and figures have all been updated.

### CODE REVIEW

**Comment by dchiu911** [comment]
Other changes:

- hyperparameter tuning for multinomial regression changed to use a 100 point random space-filling grid. No need to initiate the mixture parameter at 10 evenly spaced points from [0, 1] and then cross it with 10 random points for the penalty parameter. This ensures we traverse the space of these 2 parameters more evenly.
- Previously the validation set confusion matrices had MUOC and LGSOC true class labels mislabeled. This has now been fixed
- Fixed Equation 2.4 in the RSF to match the equation used in the supplementary regarding the 3 datasets used to construct the training set
